src/App.js

```diff
@@ -254,7 +254,7 @@ class App extends Component {
     Navigation.setTopLevelNavigator(navigatorRef);
 
   handleTransactionConfirmed = tx => {
-    const network = ethereumUtils.getNetworkFromChainId(tx.chainId);
```

Comment:
@brunobar79 should this change be separate from networking switching issues with WC?

Reply:
ideally yes but not worth the pain of pulling it out